scootapi/scoot.thrift

```diff
@@ -31,8 +35,9 @@ struct JobDefinition {
   2: optional JobType jobType,
 }
 
+// TODO(dbentley): rename this to... Exec? CloudExec? CloudScoot?
 service Proc {
-  Job RunJob(1: JobDefinition job) throws (
+   JobId RunJob(1: JobDefinition job) throws (
```

Comment:
We either need the clients to assign unique taskIds per job.  Which is fine, but it needs to be part of the API, so please update the Thrift.

We should not rely on order in the list.  That makes life much more difficult because then we have to preserve the order throughout the entire codebase.  From experience this is a pain.  A map of Id to Task info is what we want.